.taskfiles/K0sTasks.yaml

```
@@ -0,0 +1,25 @@
+---
+version: "3"
+
+tasks:
+
+  kubeconfig:
+    desc: Gets k0s cluster kubeconfig
+    cmds:
+      - k0sctl kubeconfig -c k0s-config.yaml > kubeconfig
```

```suggestion
      - k0sctl kubeconfig --cconfig k0s-config.yaml > kubeconfig
```
Nit on consistency with the tasks below this one